src/stan-users-guide/odes.Rmd

```diff
@@ -1,5 +1,54 @@
 # Ordinary Differential Equations  {#ode-solver.chapter}
 
+Stan provides three different built in methods for solving systems of
+ordinary differential equations (ODEs). If care is not taken, ODE models
+can be difficult to fit because
+
+- It is easy to make a mistake specifying the system of ODEs
+
+- A misspecified ODE can be very slow to solve -- making it difficult
+to quickly identify mistakes and interate
+
+- The numerical behavior of ODE can change radically in different parts
+of parameter space
```

Comment:
Indeed it can! I'm trying to write a poster on the subject for an upcoming conference, I'll get you involved.